_includes/_head.html

```diff
@@ -7,84 +7,71 @@
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.0/css/font-awesome.min.css">
-  {% endif %}
+<noscript>
+<link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
+</noscript>
 
-	<script src="{{ url }}/assets/js/modernizr.min.js"></script>
+<!-- Search Engine Optimization -->
+{% if page.noindex == true or layout.noindex == true %}<meta name="robots" content="noindex">{% endif %}
 
-	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
-	<script>
-		WebFont.load({
-			google: {
-				families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
-			}
-		});
-	</script>
+{% seo %}
 
-	<noscript>
-		<link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
-	</noscript>
+{% comment %} TODO: Refactor when https://github.com/jekyll/jekyll-seo-tag/pull/148 lands in 2.2.0 {% endcomment %}
+{% if site.bing_webmastertools_id %}<meta name="msvalidate.01" content="{{ site.bing_webmastertools_id }}" >{% endif %}
+{% if site.yandex_site_verification %}<meta name="yandex-verification" content="{{ site.yandex_site_verification }}">{% endif %}
+{% if site.alexa_verify_id %}<meta name="alexaVerifyID" content="{{ site.alexa_verify_id }}">{% endif %}
 
+{% comment %} TODO: Refactor image usage for jekyll-seo-tag {% endcomment %}
+{% if page.image.title %}<meta property="og:image" content="{{ site.urlimg }}{{ page.image.title }}">{% endif %}
+{% if page.image.title %}<meta name="twitter:image" content="{{ site.urlimg }}{{ page.image.title }}">{% endif %}
 
-	<!-- Search Engine Optimization -->
-	<meta name="description" content="{{ description }}">
-  	{% if site.google_site_verification %}<meta name="google-site-verification" content="{{ site.google_site_verification}}">{% endif %}
-	{% if site.bing_webmastertools_id %}<meta name="msvalidate.01" content="{{ site.bing_webmastertools_id }}" >{% endif %}
-	{% if site.google_author %}<link rel="author" href="{{ site.google_author }}">{% endif %}
-	{% if site.alexa_verify_id %}<meta name="alexaVerifyID" content="{{ site.alexa_verify_id }}">{% endif %}
-	{% if page.noindex == true or layout.noindex == true %}<meta name="robots" content="noindex">{% endif %}
-	<link rel="canonical" href="{{ url }}{{ page.url }}">
+{% assign seo_author = page.author | default: page.authors[0] | default: site.author %}
+{% if site.data.authors[seo_author] %}
+{% assign seo_author_name = site.data.authors[seo_author].name %}
+{% assign seo_author_google_plus_url = site.data.authors[seo_author].google_plus_url %}
+{% assign seo_author_facebook_user = site.data.authors[seo_author].facebook %}
+{% endif %}
 
+{% comment %} TODO: Refactor when https://github.com/jekyll/jekyll-seo-tag/issues/96 is resolved {% endcomment %}
+{% if seo_author_name %}<meta name="author" content="{{ seo_author_name }}" />{% endif %}
```

Comment:
This allows the site authors name to appear in Facebook cards when the author doesn't have a Facebook account. Here's how it works: https://github.com/jekyll/jekyll-seo-tag/issues/96#issuecomment-275840637